instaloader/structures.py

```diff
@@ -1097,6 +1097,26 @@ def get_igtv_posts(self) -> NodeIterator[Post]:
     def _make_is_newest_checker() -> Callable[[Post, Optional[Post]], bool]:
         return lambda post, first: first is None or post.date_local > first.date_local
 
+    def get_followed_hashtags(self) -> NodeIterator['Hashtag']:
+        """
+        Retrieve list of hashtags followed by given profile.
+        To use this, one needs to be logged in and private profiles has to be followed.
+
+        :rtype: NodeIterator[Hashtag]
+        .. versionadded:: 4.10
```

Comment:
```suggestion
        :rtype: NodeIterator[Hashtag]

        .. versionadded:: 4.10
```

Here a blank line is needed.

Reply:
Should be resolved now.